hack/prom-rule-ci/prom-rules-tests.yaml

```diff
@@ -672,31 +672,67 @@ tests:
 - interval: 1m
   input_series:
     - series: 'kubevirt_hco_node_status_max_images{node="node1"}'
-      values: '50+0x10 -1'
+      values: '50+0x20 -1+0x10 50+0x35'
     - series: 'kubevirt_hco_node_number_of_images{node="node1"}'
-      values: '0+10x5'
+      values: '0+0x10 51+0x55'
+    - series: 'kubevirt_hco_hyperconverged_cr_exists{}'
+      values: '1+0x40 0+0x5 1+0x20'
+    - series: 'kube_node_labels{node="node1", label_kubevirt_io_schedulable="true"}'
+      values: '1+0x60 stale'
+    - series: 'kube_node_labels{node="node1"}'
+      values: 'stale+0x60 1+0x65'
 
   alert_rule_test:
-    # Not yet 5m since kubevirt_hco_node_number_of_images reached 50 (same as kubevirt_hco_node_status_max_images)
-    - eval_time: 9m
+    # node images: 0, max images: 50, no alert
+    - eval_time: 6m
       alertname: NodeStatusMaxImagesExceeded
       exp_alerts: [ ]
 
-    # 5m since kubevirt_hco_node_number_of_images reached 50 (same as kubevirt_hco_node_status_max_images)
-    - eval_time: 10m
+    # node images: 51, max images: 50, alert
```

Comment:
You should also mention HCO CR exists

Reply:
[when HCO CR does not exist](https://github.com/kubevirt/hyperconverged-cluster-operator/pull/2274/files#diff-479cada554c66a5d0c01f01cb6c59a317d50eca03113ca6170f409d5b58b783aR722) and therefore the alert won't be triggered, I mention it specifically, on all the other tests, for readability, I think we can omit it exists